sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -331,7 +334,16 @@ private static float setDefaultConfidenceValue(Float confidence) {
      */
     private static List<FormField> toFieldValueArray(List<FieldValue> valueArray, List<ReadResult> readResults) {
         return valueArray.stream()
-            .map(fieldValue -> setFormField(null, null, fieldValue, readResults))
+            .map(fieldValue -> {
+                FieldData valueData = null;
+                // ARRAY has ho value data, such as bounding box.
+                if (ARRAY != fieldValue.getType()) {
+                    valueData = new FieldData(fieldValue.getText(), toBoundingBox(fieldValue.getBoundingBox()),
+                        fieldValue.getPage() == null ? -1 : fieldValue.getPage(),
```

Comment:
In which cases page here will be null?
For what I see in .NET, the `ContactNames` page is null so I am setting it to 1 because -1 sounds too weird.
I did create an [issue](https://github.com/Azure/azure-sdk-for-net/issues/15845) to follow up on that with the service team and I initiated a [thread](https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1602281344886?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1602281344886&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1602281344886) for that. 

If you are seeing more cases, please add to the conversation.

Reply:
We did have an offline conversation related to this. It shouldn't be null and should use fieldValue.getPage().

cc: @mssfang

Reply:
Yah. Yes. Forget this change. Will update it soon

Reply:
fieldValue.getPage() will return null, so we are assign 1 to it  as the page number?

Reply:
I think for now using `1` is ok. It might change according to what the service team says

Reply:
Instead of `1`, shouldn't we set `ContactNames` page number to whatever page number is found for `FirstName` or `LastName`? Always defaulting to `1` could be confusing if the user passes a multi-page pdf of scanned business cards. That being said, like @maririos said above, still waiting on the service team guidance on this one.